server/application-server/.env.example

```
@@ -6,4 +6,9 @@ KEYCLOAK_GITHUB_CLIENT_SECRET=replace-with-github-client-secret
 KEYCLOAK_GITHUB_ADMIN_USERNAME=replace-with-github-username-with-admin-access
 
 # GIHTUB_PAT for GitHub API authentication (as alternative for env variable)
-#GITHUB_PAT=replace-with-github-pat
\ No newline at end of file
+#GITHUB_PAT=replace-with-github-pat
+
+# Encryption key for secrets at rest (GitHub tokens, Slack tokens)
+# Required for production. If not set, secrets are stored in plaintext.
+# Generate with: openssl rand -base64 32
+#HEPHAESTUS_ENCRYPTION_SECRET_KEY=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)